### DISCUSSION TITLE
Whisper for Singapore transcriptions.

### DISCUSSION THREAD

**Discussion by meg261995** [openai/whisper#2259] [Q&A]
Could someone please clarify whether whisper understands Singapore English and Singapore Mandarin? Will it be able to recognize the accent?

**Comment by glangford** [comment]
I would recommend testing this yourself against sample(s) of audio. Speech quality, noise, accent, whisper model size and parameters can all affect the results.

Note that whisper is intended to be used on monolingual audio inputs, although it can occasionally (but not predictably) work on multiple languages. See 

https://github.com/openai/whisper/discussions/1160#discussioncomment-5586383

See also

Transcribe Singlish with OpenAI’s Whisper? “Can meh?”
https://medium.com/@nirmalya.ghosh/transcribe-singlish-with-openais-whisper-can-meh-118328e4866f

Speech Lab - AI Singapore
https://aisingapore.org/aiproducts/speech-lab/
